airbyte-webapp/src/packages/cloud/views/layout/MainView/components/CreditsProblemBanner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+import { FormattedMessage } from "react-intl";
+
+import { CreditStatus } from "packages/cloud/lib/domain/cloudWorkspaces/types";
+import { Link } from "components/Link";
+import { CloudRoutes } from "packages/cloud/cloudRoutes";
+
+const Container = styled.div`
+  height: 30px;
+  width: 100%;
+  background: ${({ theme }) => theme.redColor};
+  color: ${({ theme }) => theme.whiteColor};
```

Comment:
We need to use black text on that background color to achieve the proper (see https://color.review/check/FFFFFF-FF6A4D).